R/interpolate.R

```diff
@@ -29,18 +29,17 @@ setGeneric("sqlInterpolate", function(`_con`, `_sql`, ...) {
 setMethod("sqlInterpolate", "DBIConnection", function(`_con`, `_sql`, ...) {
   sql <- `_sql`
   pos <- sqlParseVariables(`_con`, sql)
-
+  
```

Comment:
Nitpick... Current RStudio trims whitespace, I've updated the project settings in d2c7a78f90d21385860adba04b4bd72f19e566fb.